test/libsolidity/semanticTests/storage/mappings_array2d_pop_delete.sol

```diff
@@ -0,0 +1,39 @@
+contract C {
+    mapping (uint => uint)[][] a;
+
+	function n1(uint key, uint value) public {
+		a.length++;
+		mapping (uint => uint)[] storage b = a[a.length - 1];
+		b.length++;
+		b[b.length - 1][key] = value;
+	}
```

Comment:
Note to self: This creates an array of mappings of dimensions `[1][1]` or (in other words) a single mapping of uint -> uint and initializes the mapping to m[42] = 64;